src/config/database/configuration.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { TypeOrmModuleOptions, TypeOrmOptionsFactory } from '@nestjs/typeorm';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class DatabaseConfiguration implements TypeOrmOptionsFactory {
+  constructor(private configService: ConfigService) {}
+
+  createTypeOrmOptions(): TypeOrmModuleOptions {
+    return {
+      type: 'mysql',
```

Comment:
can we get this also from env?

Assuming that we will later on not worry about which database we use in the background.

Reply:
Trying to create and fetch the `type` value from `.env` file results in the following error:
![image](https://github.com/OsmosysSoftware/osmo-notify/assets/108812833/1836686a-229f-4b95-8a95-7cef7faf3ab0)